### ISSUE TITLE
Add "Data Integration" link to site headers

### ISSUE THREAD

**Issue by jkelliher-github** [microbiomedata/issues#778]
Created new webpage to feature DOE BER data integration. Webpage available here: https://microbiomedata.org/data-integration/ 
Needs to be added to the main website menu under 'Resources' (at the bottom of the Resources tab is fine) with the title 'Data Integration'. This will be included in the newsletter scheduled to be sent out today.

Thanks so much!

Also weird thing, if you go into the wordpress page editor (at least on my side), I am not seeing any of the actual content that I put into the webpage... is this a known issue or am I just missing a button to show me the content?

**Comment by pkalita-lbl** [comment]
The Data Integration link has been added to the header of the WordPress site.

I will open a PR to get the portal's header updated later today. Those changes will roll out with the production deployment scheduled for July 29.

> Also weird thing, if you go into the wordpress page editor (at least on my side), I am not seeing any of the actual content that I put into the webpage... is this a known issue or am I just missing a button to show me the content?

I'm not sure I see what you're describing. When I've logged into the WP Admin site and I visit https://microbiomedata.org/data-integration/ I can see the page content there. I clicked on "Edit Page" in the admin banner at the top of the page to get to the Edit Page screen. I can also see the content there.
